app/spec/lib/bk/compat/snapshots/examples/circleci/install-docker-tools.yml.snap

```diff
@@ -0,0 +1,81 @@
+---
+steps:
+- commands:
+  - "# No need for checkout, the agent takes care of that"
+  - "# Instead of installing docker in a step, "
+  - "# we recommend your agent environment to have it pre-installed"
+  - echo '~~~ Installing Docker'
+  - curl -fsSL https://get.docker.com -o get-docker.sh
+  - sh get-docker.sh
+  - "# Instead of installing docker-compose in a step, "
+  - "# we recommend your agent environment to have it pre-installed"
+  - echo '~~~ Installing docker-compose'
+  - PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]')
+  - SYS_ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
+  - SYS_ARCH=${SYS_ARCH/arm64/aarch64}
+  - version="latest"
+  - if [ "$version" = "latest" ]; then
+  - "  RELEASE_PATH=latest/"
+  - else
+  - "  RELEASE_PATH=download/${version}/"
+  - fi
+  - BINARY_URL=https://github.com/docker/compose/releases/${RELEASE_PATH}docker-compose-${PLATFORM}-${SYS_ARCH}
+  - curl -L ${BINARY_URL} -o /usr/local/bin/docker-compose
+  - chmod +x /usr/local/bin/docker-compose
```

Comment:
Per the original YAML file all tools should be installed in `/var/lib/docker` but none of them are